### DISCUSSION TITLE
Using AWS-Java-SDK-S3, return to Inform Error 501 Not Implemented when uploading files

### DISCUSSION THREAD

**Discussion by fantastic2085** [aws/aws-sdk-java#2819] [Q&A] [answered]
**Upload files to the CEPH object store using AWS-java-SDK-S3**

s3 client The following code:
<img width="759" alt="b9fec4281e334f985164713cde4ab76" src="https://user-images.githubusercontent.com/4116773/182992056-0a1f4959-3417-4336-87a4-c491679bc2d5.png">

AWS-Java-SDK-S3 version:
<img width="258" alt="47b9a594d6941fc3df6a08d307a2777" src="https://user-images.githubusercontent.com/4116773/182992114-e2c3cbe3-461e-41c3-8594-eca099bded62.png">

The code to upload the file is as follows:
![image](https://user-images.githubusercontent.com/4116773/182992523-051a630d-b9fa-4e77-b2b9-8facc89225dc.png)

An error is as follows:
<img width="755" alt="a88dd29acca63f410c4aa96d5fcee31" src="https://user-images.githubusercontent.com/4116773/182992303-e142757a-5007-4586-a8f1-902e19bc2422.png">

**Comment by yasminetalby** [comment, marked as answer]
Hello @fantastic2085 ,

You have made a duplicate submission as a bug. I will respond to you over [there](https://github.com/aws/aws-sdk-java/issues/2820) to prevent any confusion.

Best,

Yasmine
